lib/qs-middleware.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 
-var qs = require('qs');
 var url = require('url');
+var qs = require('qs');
 
 module.exports = qsMiddleware;
 
 function qsMiddleware(options) {
-	return function(request, response, next) {
+	return function (request, response, next) {
```

Comment:
It's interesting that it wants a space after `function`, here, but not after `qsMiddleware`, the line above.  I guess it makes sense as there's also a space before `qsMiddleware`.